lms/static/sass/discussion/_edx-pattern-library-loader.scss

```diff
@@ -0,0 +1,23 @@
+// Discussions - CSS application architecture
+// Version 1 styling (pre-Pattern Library)
+// ====================
+
+// Load the LTR version of the edX Pattern Library
```

Comment:
Nit: The "LTR" part of this comment is not applicable here.. I don't think the code below is LTR specific?